packages/api-explorer/src/components/JsonViewer/index.jsx

```diff
@@ -41,11 +42,14 @@ export default function JsonViewer({ schema, missingMessage }) {
           }}
         />
       </div>
-    ) : <FormattedMessage id={missingMessage} />
+    ) : message
   )
 }
 
 JsonViewer.propTypes = {
-  schema: PropTypes.object,
+  schema: PropTypes.oneOfType([
+    PropTypes.object,
+    PropTypes.array
```

Comment:
When a schema could be an array?

Reply:
AnyOf case

Reply:
It's not an object with a `anyOf` key and an array as children?

Reply:
In this case the example can return and array

![Schermata 2020-05-21 alle 11 07 16](https://user-images.githubusercontent.com/22480239/82544165-ffd61600-9b54-11ea-8a61-879d2ddc4dac.png)

Reply:
![Schermata 2020-05-21 alle 11 21 04](https://user-images.githubusercontent.com/22480239/82544269-2e53f100-9b55-11ea-895e-45500ace2646.png)

Reply:
this question of yours made me think that perhaps the management of the example scheme was incorrect. I reset the Proptypes.object